snmp_v3/models.py

```diff
@@ -22,13 +24,23 @@ class SnmpConfiguration(models.Model):
     username = models.CharField(max_length=100)
     authentication_password = models.CharField(max_length=100)
     privacy_password = models.CharField(max_length=100)
-    service = models.ForeignKey(Service, related_name="snmp_configuration", on_delete=models.CASCADE)
+    service = models.ForeignKey(Service, related_name="snmp_configurations", on_delete=models.CASCADE)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
 
     @property
     def error_percentage(self):
-        return ErrorPercentage(hour=1, day=1, week=1)
```

Comment:
Umm... to w końcu miesiąc, tydzień, dzień czy 
![image](https://user-images.githubusercontent.com/37639712/68598435-16612d80-049f-11ea-9044-576d6634bef9.png)
Może jednak lepsze będzie tydzień-dzień-godzina, bo jak ktoś sprawdza apkę 3 razy w tygodniu to nie potrzebuje danych z całego miesiąca (przynajmniej dla pinga).